crates/aptos-rosetta/src/lib.rs

```diff
@@ -48,6 +51,7 @@ pub struct RosettaContext {
     pub coin_cache: Arc<CoinCache>,
     /// Block index cache
     pub block_cache: Option<Arc<BlockCache>>,
+    pub accounts: Arc<Mutex<BTreeMap<AccountAddress, u64>>>,
```

Comment:
Could you add a comment explaining what this u64 is? Or better, is there a seqnum newtype we can use?

Reply:
Sure